Kernel/Devices/TTY/TTY.cpp

```diff
@@ -87,14 +87,14 @@ ErrorOr<size_t> TTY::write(OpenFileDescription&, u64, UserOrKernelBuffer const&
 
     constexpr size_t num_chars = 256;
     return buffer.read_buffered<num_chars>(size, [&](ReadonlyBytes bytes) -> ErrorOr<size_t> {
-        u8 modified_data[num_chars * 2];
+        auto modified_data = Array<u8, num_chars * 2>::from_repeated_value(0);
```

Comment:
Doesn't `Array<u8, num_chars * 2> modified_data {};` give you zero initialization too?

Reply:
Not sure? How can I know for sure?

Reply:
Like this: https://godbolt.org/z/8cneW5vrj

Or, alternatively, by studying standard:
- List-initialization will delegate to aggregate initialization here: https://eel.is/c++draft/dcl.init.list#3.4
- ... which will copy-initialize u8 array inside AK::Array with `{}` here: https://eel.is/c++draft/dcl.init.aggr#5.2
- ... which delegates to value-initialization here: https://eel.is/c++draft/dcl.init.list#3.11
- ... which does value-initialization for all elements of that array here: https://eel.is/c++draft/dcl.init.general#9.2
- ... which are zeroed as per https://eel.is/c++draft/dcl.init.general#9.3.

Reply:
OK, thanks for helping! I will rebase this and fix things accordingly :)